ci/publish-crate.sh

```diff
@@ -35,8 +43,22 @@ for Cargo_toml in $Cargo_tomls; do
     # so use the solana rust docker image until this is resolved upstream
     source ci/rust-version.sh
     ci/docker-run.sh "$rust_stable_docker_image" bash -exc "cd $crate; $cargoCommand"
-    #ci/docker-run.sh rust bash -exc "cd $crate; $cargoCommand"
   ) || true # <-- Don't fail.  We want to be able to retry the job in cases when a publish fails halfway due to network/cloud issues
+
+  crate_name=$(grep -m 1 '^name = ' $Cargo_toml | cut -f 3 -d ' ' | tr -d \")
+  numRetries=30
+  sleepTime=2
+  for ((i = 1 ; i <= $numRetries ; i++)); do
+    echo "Attempt ${i} of ${numRetries}"
+    if $(is_crate_version_uploaded $crate_name $expectedCrateVersion) = "True"
+    then
+      echo "Found ${crate_name} version ${expectedCrateVersion} on crates.io"
+      break
+    else
```

Comment:
nit: `else` block is not necessary due to the `break`.   Instead just close with a `fi` here and unindent the next two lines